modules/integration/tests-common/admin-clients/src/main/java/org/wso2/bps/integration/common/clients/bpmn/ActivitiRestClient.java

```diff
@@ -63,7 +63,7 @@
     public ActivitiRestClient(String portM, String hostnameM) {
         port = Integer.parseInt(portM);
         hostname = hostnameM;
-        serviceURL = "http://" + hostnameM + ":" + portM + "/bpmnrest/";
+        serviceURL = "http://" + hostnameM + ":" + portM + "/bpmn/";
```

Comment:
Changed variable type from string to URL. Please PR https://github.com/wso2/product-bps/pull/43>

Thank you.